docs/install.md

```diff
@@ -335,35 +335,35 @@ Select the following tabs for the installation description of your target.
 
     <font size="+2">1. Generate a new Proxmox VM</font>
 
-    - Access the Proxmox web interface via HTTPS on TCP port **8006**:
+    1. Access the Proxmox web interface via HTTPS on TCP port **8006**:
         - URL: `https://<your.IP>:8006`
         - Username: `root`
         - Password: `<root user password you entered during Proxmox VE install>`
-    - Select the **Create VM** button at the top right corner. Choose a **Node** , a **VM ID** and a **Name**, then click **Next**.
+    1. Tab **General**: Select the **Create VM** button at the top right corner. Choose a **Node** , a **VM ID** and a **Name**, then click **Next**.
 
-        ![Proxmox VM creation](assets/images/proxmox2.png){: width="722" height="314" loading="lazy"}
+        ![Proxmox VM creation](assets/images/proxmox2.png){: width="600" height="260" loading="lazy"}
```

Comment:
Note that the image width is adjusted automatically to fit into the container. Not sure what the benefit is to reduce it manually in this case? Scaling always implies worse quality, and especially on large screens we should make use of the full image size, keeping it readable and not loosing quality.

Reply:
But this highly depends on screen/browser window width. In your particular case it may look nicer, but on larger screens it looks much worse. So generally I'd vote for using the native size to keep the quality and readability of the image at least on larger screens. Of course we could use a smaller crop of the image, but this frame cannot be made smaller, so we'd need to crop off parts left and/or right, which to me doesn't look nice either.

Reply:
I checked back the style on different screen sizes. Actually the maximum content width (between nav and TOC) is 883 pixels on screens > 1920 (full HD), with . On 1920px screens it's 767px, on smaller screens, first the nav, then the TOC is collapsed, allowing a content width of max 962px at exactly 1219px browser window width.

It is quite easy to apply a max-width for images of 95%, so they are not wider than text (considering line breaks with full words). So to have non-resized images on most screen, we shouldn't use images larger than 725px native size, where possible. Even on >full HD screens the image cannot be that much larger.

I'll apply the 95% max-width rule, so we see whether the layout looks fine with it, without manually capping the image dimensions.